src/main/java/io/appium/java_client/AppiumDriver.java

```diff
@@ -291,7 +291,6 @@ protected void startSession(Capabilities capabilities) {
 
         @SuppressWarnings("unchecked") Map<String, Object> rawCapabilities = (Map<String, Object>) responseValue;
         // A workaround for Selenium API enforcing some legacy capability values
-        rawCapabilities.remove(CapabilityType.PLATFORM);
```

Comment:
is the workaround not needed anymore?

Reply:
`CapabilityType.PLATFORM` is removed from Selenium, so I guess Selenium doesn't enforce any capability named "platform"

Reply:
also do not forget to remove the obsolete comment above after you verify the capability is not there anymore